config/global/paths.in

```diff
@@ -133,3 +133,31 @@ config STRIP_TARGET_TOOLCHAIN_EXECUTABLES
       An install-strip make target is provided that installs stripped
       executables, and may install libraries with unneeded or debugging
       sections stripped. 
+
+config TARBALL_RESULT
+    bool
+    depends on EXPERIMENTAL
+    prompt "Create binary toolchain tarball"
+    default n
+    help
+      Create tarball of the final binary toolchain.
+
+if TARBALL_RESULT
+
+config TARBALL_RESULT_DIR
+    string
+    depends on TARBALL_RESULT
+    prompt "Output directory"
+    default "${CT_TOP_DIR}"
+    help
+      Directory where tarball will be created.
+
+config TARBALL_RESULT_FILENAME
+    string
+    depends on TARBALL_RESULT
+    prompt "Output filename"
+    default "toolchain-${CT_TOOLCHAIN_PKGVERSION}-${CT_HOST:+HOST-${CT_HOST}-}${CT_TARGET}"
```

Comment:
CT_TOOLCHAIN_PKGVERSION is empty by default. This would create an odd name with two dashes in a row.

`"toolchain${CT_TOOLCHAIN_PKGVERSION:+-${CT_TOOLCHAIN_PKGVERSION}}-${CT_HOST:+HOST-${CT_HOST}-}${CT_TARGET}"`

perhaps?